src/js/media-controller.js

```diff
@@ -292,6 +319,11 @@ class MediaController extends MediaContainer {
         stateDetails.get(this)
       );
     });
+    if (this._mediaStore) {
+      Object.entries(this._mediaStore.getState()).forEach(([stateName, stateValue]) => {
+        propagateMediaState([el], stateName.toLowerCase(), stateValue);
```

Comment:
Is this just a simplified implementation or a choice to move to pushing all state to a stateReceiver, not just what it cares about? Or does propagateMediaState just work that out under the hood before pushing to the receiver?

Reply:
The last one, but I think we could consider simplifying that as well (since it's all "under the hood", not a blocker for v1.0 imo)